Community Guidelines (#1)
typicalbot/policies: Community_Guidelines.md

# Community Guidelines

> In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that we allow. Every member is expected to adhere to the guidelines set forth in these Community Guidelines. Failure to do so may result in the temporary or permanent banning of your account.

Reviewer: Could we get a "last updated" added under this, preferably in italics. Such as:

_Last updated: March 22, 2019_

Reviewer: In fact, it might also be useful to have a link next to that says "See archives", or something similar.

> In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that we allow. Every member is expected to adhere to the guidelines set forth in these Community Guidelines. Failure to do so may result in the temporary or permanent banning of your account.

Reviewer 2: ```suggestion
In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that we allow. Every member is expected to adhere to the Community Guidelines set forth in this document. Failure to do so may result in the temporary or permanent banning of your account.
```

Reviewer 3: The "Community Guidelines" is a proper noun that refers to this specific document, in this instance. The "guidelines" refers to each of the different rules. As such, I don't believe this suggested change is appropriate.

> In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that we allow. Every member is expected to adhere to the guidelines set forth in these Community Guidelines. Failure to do so may result in the temporary or permanent banning of your account.

Author: ```suggestion
In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that is permitted. Members are expected to adhere to the guidelines set forth in these Community Guidelines. Failure to do so may result in the temporary or permanent banning of your account.
```
Some grammar and stylistic stuff. What do you think?

Reviewer 3: Not opposed, but the wording is important here. "Every member" is more powerful than "members", for example. It clarifies any misunderstandings about what members are being referred to.

---

## Advertising

> Do not advertise in our community. By advertise, we mean send messages (in our server or by direct message) or use a nickname that are designed to promote something that isn't in connection with TypicalBot.

Reviewer 4: ```suggestion
Do not advertise in our community. By advertise, we mean send messages (in our server or by direct message to community members) or use a nickname that are designed to promote something that isn't in connection with TypicalBot.
```

Reviewer 3: I made updates to the advertising guidelines, which clarify that advertising in the server or to our members in any way is not allowed.

## Backseat Moderating

> Do not "backseat moderate". "Backseat moderating" means acting like or taking on the role of a moderator.

Reviewer: Beneath this, we should add a rule about impersonation? _It has happen before..._

> ## Disrespect and Harassment

Reviewer: Let's widen this topic to cover inappropriate language/behavior.

Reviewer 3: Anything specifically with regard to inappropriate behavior?

Reviewer: I would think that anything considered offensive, racist or obscene should not be said.

> Do not disrespect or harass anyone. This includes, but isn't limited to, making racist or obscene comments towards other members of the community and using derogatory or vulgar language in a manner that's intended to disrespect or harass another member.

Reviewer 2: ```suggestion
Do not disrespect or harass anyone. This includes, but is not limited to, making racist or obscene comments towards other members of the community and/or using derogatory or vulgar language in a manner that's intended to disrespect or harass another member.
```

## Impersonation

> Do not impersonate any person or group of people. Do not act in a manner that's intended to mislead others into believing that you are someone that you are not. 

Reviewer 2: ```suggestion
Do not impersonate any person or group of people. Do not act in a manner that is intended to mislead others into believing that you are someone that you are not. 
```

> Do not impersonate any person or group of people. Do not act in a manner that is intended to mislead others into believing that you are someone that you are not. 

Author: I don't think the phrase "into believing that you are someone that you are not" is necessary. It seems redundant in the context of the rule.

Reviewer 3: It's meant to add clarification and double-down on the rule. Probably not necessary, but misleading isn't necessarily impersonation, and I wanted to clarify that both aren't okay.

## Mentions
Do not mention roles on the server, even if a role is set as mentionable. Do not mention a member excessively.

## Bugs

> Do not exploit bugs or issues with Discord or any of the bots on our server.

Reviewer: Maybe we should change this to use "knowingly exploiting a bug"?

> Do not ask for or request a role on our server. Additionally, do not mention a role on our server, even if it's mentionable. If you are in need of support, you can mention the @Support role, but we ask that you use this wisely. Members should not be mentioned excessively.

Reviewer: Could we make the "@Support" inline, by using backticks? Such as: `@Support`

> Do not ask for or request a role on our server. Additionally, do not mention a role on our server, even if it's mentionable. If you are in need of support, you can mention the `@Support` role, but we ask that you use this wisely. Members should not be mentioned excessively.

Reviewer 2: ```suggestion
Do not ask for or request a role on our server. Additionally, do not mention any roles on our server, even if it is mentionable, with the exception of the support role. If you are in need of support, you can mention the `@Support` role, but we ask that you use this wisely. Members should not be mentioned excessively.
```

## Spam
Do not post spam. Our definition of "spam" includes unsolicited advertising, [copypastas](https://www.urbandictionary.com/define.php?term=copypasta) and false PSAs. We also do not permit trolling of any kind in our community.

## NSFW

> Do not use vulgar language excessively. Do not post any messages that contain content that is NSFW. NSFW content includes, but is not limited to, pornography, violence and other distributing or objectionable content. Content that is prohibited by the [Discord Community Guidelines](https://discordapp.com/guidelines) is strictly forbidden in our community.

Author: ```suggestion
Do not use vulgar language excessively. Do not post any messages that contain content that is NSFW. NSFW content includes, but is not limited to, pornography, violence, and other disturbing or objectionable content. Content that is prohibited by the [Discord Community Guidelines](https://discordapp.com/guidelines) is strictly forbidden in our community.
```
Changed "distributing" to "disturbing" and added a comma- though the 'Oxford Comma' isn't necessary, it improves comprehension.

# Community Guidelines
In order to protect the experience and safety of the members of our community, we restrict the type of behavior and content that we allow. Every member is expected to adhere to the guidelines set forth in these Community Guidelines. Failure to do so may result in the temporary or permanent banning of your account.

---

## Advertising

> Do not advertise in our server or to our members. Advertising means sending messages or using other features of Discord in a manner designed to promote something that isn't related to the projects of the community.

Reviewer 5: I would specify what "related" is here, as otherwise, someone will advertise their own bot guild due to both projects being related through Discord.

## Backseat Moderating

> Do not "backseat moderate". "Backseat moderating" means acting like or taking on the role of a moderator.

Reviewer 5: "means" should be changed to "is defined as"

Reviewer 3: "Means" is acceptable in this context.

## Behavior and Conduct

> Do not disrespect or harass anyone and do not make racist or obscene comments towards other members of the community. This includes, but is not limited to, using derogatory or vulgar language in a manner that's intended to disrespect or harass another member.

Reviewer 5: "and do not" should be "nor" for brevity and to prevent repetition.

## Impersonation
Do not impersonate any person or group of people. Do not act in a manner that's intended to mislead others into believing that you are someone that you are not.

## Roles
Do not ask for or request a role on our server. Additionally, do not mention a role on our server, even if it's mentionable. If you are in need of support, you can mention the @Support role, but we ask that you use this wisely.

## Mentions

> Do not mention roles on the server, even if a role is set as mentionable. Do not mention a member excessively.

Reviewer 5: This rule seems redundant. Either remove this clause from the "Roles" section or remove this clause entirely.

## Bugs
Do not intentionally exploit bugs or issues with Discord or any of the bots on our server. Knowingly exploiting bugs in Discord or any of our bots in order to cause harm will result in a permanent ban from our community.

## Spam

> Do not post spam. Our definition of "spam" includes unsolicited advertising, [copypastas](https://www.urbandictionary.com/define.php?term=copypasta) and false PSAs. We also do not permit trolling of any kind in our community.

Reviewer 5: Maybe define trolling?

Reviewer 3: I think that is better left up to the moderator, since the definition of a troll could evolve over time.

## NSFW

> Do not use vulgar language excessively. Do not post any messages that contain content that is NSFW. NSFW content includes, but isn't limited to, pornography, violence and other distribuing or objectionable content. Content that is prohibited by the [Discord Community Guidelines](https://discordapp.com/guidelines) is strictly forbidden in our community.

Reviewer 5: It might also be worthwhile to bring up the Discord Partner code of conduct for further justification of this rule.

Reviewer 3: The Discord Partner Code of Conduct is between partners and Discord. The user is not a party to the Discord Partner Code of Conduct, so it wouldn't make sense to reference it here.

## Contacting Staff

> If you have a question for staff, please use the server to communicate with them. Do not send a direct message to staff unless they ask you to do so. If you find a security issue with our bots or website, please report it to us responsibily.

Reviewer 5: "responsibily" should be spelt as "responsibly". Also, it might be a good idea to describe how to report security issues in this clause.

Reviewer: I can confirm that security issues should be emailed to security@typicalbot.com.
